### ISSUE TITLE
Hint buttons in messages

### ISSUE THREAD

**Issue by tulir** [matrix-org/matrix-doc#1485]
Documentation: https://docs.google.com/document/d/1806FWv2B8_vca5ggJC_YGKgz7npbOy7Mg_9LyP74_0w/edit#

Based on riot web PR by @robinhood23rus (matrix-org/matrix-react-sdk#2109)

**Comment by MurzNN** [comment]
Will button actions (clicks) be based on reactions, described in #441, or separate event type?

**Comment by tulir** [comment]
No, they're not reactions. Though they would probably be aggregated if aggregations work like they're described in the latest proposal (#1849)

**Comment by richvdh** [comment]
This sounds like it would fix #824?

**Comment by tulir** [comment]
Yes

**Comment by the0d0re9** [comment]
Hey, what's the status on this proposal? It sounds super useful, unsure if people have just forgotten about it, or decided it was a bad idea or something
